src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -102,6 +125,7 @@ const FluxEditorMonaco: FC<Props> = ({
 
   return (
     <ErrorBoundary>
+      <div id={ICON_SYNC_ID} style={{zIndex: 999, position: 'absolute'}} />
```

Comment:
This is the invisible div, which will be re-positioned in `_setCompositionSyncToggle()`